.circleci/config.yml

```diff
@@ -1,88 +1,68 @@
 version: 2.1
 
 orbs:
-  sonar: hubci/sonar@1.0.0
+  cimg: circleci/cimg@0.3.0
 
 workflows:
-  main:
+  main-wf:
     jobs:
-      - build:
+      - cimg/build-and-deploy:
+          name: "Staging"
+          docker-namespace: ccitest
+          docker-repository: postgres
+          publish-branch: test
+          filters:
+            branches:
+              ignore:
+                - main
+          post-steps:
+            - run:
+                name: Install Prerequisites
+                command: |
+                  sudo apt-get update && \
+                  sudo apt-get install -y postgresql-client
+            - run:
+                name: Test Docker Images
+                shell: /bin/bash -o pipefail
+                command: |
+                  ssh -f -N -L localhost:5432:localhost:5432 remote-docker
+                  IMAGES=$(docker images --format='{{.Repository}}:{{.Tag}}' | grep "ccitest/postgres")
+                  for IMAGE in $IMAGES; do
+                    printf "Booting $IMAGE...\n"
+                    CONTAINER_ID=$(docker run --rm --env POSTGRES_USER=user --env POSTGRES_PASSWORD=passw0rd -p 5432:5432 -d $IMAGE)
+                    sleep 60s
```

Comment:
Since we have `shell: /bin/bash -o pipefail` we shouldn't need this sleep anymore